client-src/live/webpack.config.js

```diff
@@ -33,5 +34,11 @@ module.exports = {
         to: path.resolve(__dirname, '../../client/live.html'),
       },
     ]),
+    new webpack.NormalModuleReplacementPlugin(/\/clients\//, (resource) => {
```

Comment:
memo: We will delete this plugin at the next major version because we can change the directory structure.